wf-galleon/WFLY-19021-Stability_In_Provisioning.adoc

```diff
@@ -0,0 +1,165 @@
+= WildFly provisioning to support WildFly stability
+:author:            Jean-Francois Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+WildFly 31 introduced the notion of stability level. Galleon, WildFly Galleon plugins and provisioning tooling must be evolved to support this notion.
+This proposal describes how stability is handled by the WildFly provisioning. 
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-19021[WFLY-19021] - WildFly provisioning to support WildFly stability
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFCORE-6668[WFCORE-6668] - Ensure read-feature-description results contain stability as reported by associated description for use by WildFly Glow
+* https://issues.redhat.com/browse/WFGP-265[WFGP-265] - Support setting a minimum stability level when provisioning
+* https://issues.redhat.com/browse/WFGP-264[WFGP-264] - Support setting a minimum stability level when building a feature pack
+* https://issues.redhat.com/browse/GAL-357[GAL-357] - Support for feature/package stability
+
```

Comment:
Please add a 'Stability Level' section as illustrated in https://raw.githubusercontent.com/wildfly/wildfly-proposals/main/design-doc-template.adoc.

You need to pick a level, but I'm pretty sure it has to be 'Community'. Definitely not 'Default'. I suppose we could say 'Preview', but it feels a bit odd to use 'Preview' functionality to generate fps and provision servers at a higher level. It's not formally wrong though.

'Community' will come with stricter requirements, e.g. around testing.

Reply:
Whatever you pick, please preface the line 1 heading and the PR title with it, in braces, e.g. 

[Community] WildFly provisioning to support WildFly stability

Proposals now should all be about doing X at level Y and the top level heading should declare Y in a standard way.

Reply:
Done.